tests/pl/_fixtures.yaml

```diff
@@ -495,118 +495,118 @@ entities:
       device_class: window
 
   # https://www.home-assistant.io/integrations/sensor/
-  - name: "Moc pozorna urządzenia"
+  - name: "Moc pozorn[a|ą] urządzenia"
     id: "sensor.appliance_apparent_power"
     state: "2"
     attributes:
       device_class: apparent_power
       unit_of_measurement: "VA"
 
-  - name: "Czujnik powietrza w kuchni"
+  - name: "Czujnik[u|a] powietrza w kuchni"
     id: "sensor.kitchen_aqi_sensor"
     state: "50"
     attributes:
       device_class: aqi
 
-  - name: "Czujnik ciśnienia"
+  - name: "Czujnik[u] ciśnienia zewnętrznego"
     id: "sensor.outside_pressure"
     state: "1000"
     attributes:
       device_class: atmospheric_pressure
       unit_of_measurement: "hPa"
 
-  - name: "Mój telefon"
+  - name: "Mój telefon|Moim telefonie|Mojego telefonu"
     id: "sensor.my_phone_battery"
     state: "98"
     attributes:
       device_class: battery
       unit_of_measurement: "%"
 
-  - name: "Dwutlenek węgla"
+  - name: "Pomp(a|y|ę|ie) ciepła" # Dwutlenek węgla
     id: "sensor.heat_pump_co2"
     state: "50"
     attributes:
       device_class: carbon_dioxide
       unit_of_measurement: "ppm"
 
-  - name: "Tlenek węgla"
+  - name: "Pomp(a|y|ę|ie) ciepła" # Tlenek węgla
```

Comment:
Isn't this a problem that we now have two identical ones here? If we ask for it which one should be returned?

Reply:
This is not a problem, because we will check specific `device_class` for this name. In one case we use `carbon_dioxide` but in the other `carbon_monoxide`.